components/brave_rewards/resources/ui/components/rewardsButton/style.ts

```diff
@@ -47,7 +47,7 @@ const getTypeStyle = (type?: Type, disabled?: boolean) => {
           background: ${p =>
             disabled
               ? p.theme.color.brandBrave
-              : p.theme.brandBraveInteracting
+              : p.theme.color.brandBraveInteracting
```

Comment:
Another nice find 🙂 Guessing this is broken right now in release? Is there an easy way to audit for items like this which may not resolve?

Reply:
ah nice! that's perfect 😄